File: Arrays/LinearSearch/ArraySearch.c
#include<stdio.h>

int findElement(int arr[],int n,int key)
{
    int i;
    for(i=0;i<n;i++)
    {
        if(arr[i]==key)
        {
            return i;
        }
    }
    return -1;
}
int main()
{
    int arr[]={2,3,45,1,21};
    int n=sizeof(arr)/sizeof(arr[0]);
    int key =21;
    int position=findElement(arr,n,key);
    if(position==-1)
    {
        printf("Element Not found");
    }
    else{
        printf("Element Found at Position:%d",position+1);
    }
    return 0;
}

//Our put:Element Found At Position:5

File: Arrays/LinearSearch/ArraySearch.js
function findElement(arr,n,key)
{
    let i;
    for(i=0;i<n;i++)
    {
        if(arr[i]==key)
        {
            return i;
        }
    }
    return -1;
}
let arr=[1,2,3,4,43,23];
let n=arr.length;
let key=23;
let position=findElement(arr,n,key);
if (position == - 1)
    console.log("Element not found");
else
    console.log("Element Found at Position: " 
         + (position + 1));